app/src/main/java/com/medium/highlightjs/demo/EditActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.pddstudio.highlightjs.demo
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.widget.doOnTextChanged
+import com.medium.highlightjs.demo.databinding.ActivityEditBinding
+import com.medium.highlightjs.models.Theme
+
+
+class EditActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityEditBinding
+
+    private val source = """public class MainActivity extends AppCompatActivity {
```

Comment:
Should this be deleted?

Reply:
I like to keep it when I want to try different config